jerry-core/vm/vm.c

```diff
@@ -328,6 +328,128 @@ vm_get_implicit_this_value (ecma_value_t *this_value_p) /**< in-out: this value
 } /* vm_get_implicit_this_value */
 
 /**
+ * 'Function call' opcode handler.
+ *
+ * See also: ECMA-262 v5, 11.2.3
+ */
+static void
+opfunc_call (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
```

Comment:
Some opfuncs are part of vm.c because opcodes.c does not depend on byte code format. We could refactor this but I would do it in a seperate patch.